pkg/services/app/add.go

```diff
@@ -851,7 +851,9 @@ func (a *AppResourceInfo) appResourceName() string {
 }
 
 func (a *AppResourceInfo) appSecretName(repoURL string) string {
-	return fmt.Sprintf("wego-%s-%s", a.targetName, utils.UrlToRepoName(repoURL))
+	repoName := utils.UrlToRepoName(repoURL)
+	repoName = strings.ReplaceAll(repoName, "_", "-")
```

Comment:
Can you write a unit test for this function?

Reply:
It might be worth make a small utility function called SanitizeSecretName in the `utils` package.  There are other places we might want to use this functionality in the future.